cores/esp8266/sntp-lwip2.cpp

```diff
@@ -65,7 +65,7 @@ extern "C" {
 
 static const char stod14[] PROGMEM = "settimeofday() can't set time!\n";
 bool sntp_set_timezone(sint8 timezone);
-bool sntp_set_timezone_in_seconds(sint32 timezone)
+bool sntp_set_timezone_in_seconds(uint32_t timezone)
```

Comment:
I'm no expert o/n timezones, but I'm pretty sure UTC-8 exists.  Wouldn't this change make it impossible to set negative UTC timezones